event_contact/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Event Contacts
+==============
+
+This module adds the possibility to define contacts for
+the event. Contacts are needed due to inform the people
+who should be contacted for the event
+
+Installation
```

Comment:
Remove this obvious instructions that are also referred to 8.0 version (and other people can obtain the module via Apps).